cmd/kueue/main.go

```diff
@@ -226,7 +226,7 @@ func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manag
 	// setup provision admission check controller
 	if features.Enabled(features.ProvisioningACC) && provisioning.ServerSupportsProvisioningRequest(mgr) {
 		// A info message is added in setupIndexes if autoscaling is not supported by the cluster
-		if err := provisioning.NewController(mgr.GetClient()).SetupWithManager(mgr); err != nil {
+		if err := provisioning.NewController(mgr.GetClient(), mgr.GetEventRecorderFor("admission-checks-controller")).SetupWithManager(mgr); err != nil {
```

Comment:
The name should be specific to ProvReq.

Let's say `kueue-provisioning-request-controller`.

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1271/commits/3b15bd9a0ad460731328c4e0eaeb91897f09cfb0